packages/website/docs/reverseHighlightHit.md

```diff
@@ -2,36 +2,84 @@
 id: reverseHighlightHit
 ---
 
-Returns a virtual node with non-matching parts of an Algolia hit.
+Returns a virtual node with highlighted non-matching parts of an Algolia hit.
 
-## Example
+ The `reverseHighlightHit` function lets you turn an Algolia hit into a virtual node with highlighted non-matching parts for a given attribute.
+
+## Examples
+
+### With a single string
+
+To determine what attribute to parse, you can pass it as a string.
 
 ```js
 import { reverseHighlightHit } from '@algolia/autocomplete-js';
 
-const hit = {}; // fetch an Algolia hit
-const highlightedValue = reverseHighlightHit({
+// An Algolia hit for query "hello"
+const hit = {
+  query: 'Hello there',
+  _highlightResult: {
+    query: {
+      value:
+        'Hello __aa-highlight__there__/aa-highlight__',
+    },
+  },
+};
+const reverseHighlightedValue = reverseHighlightHit({
   hit,
   attribute: 'query',
 });
 ```
 
-## Params
+### With nested attributes
+
+If you're referencing a nested attribute, you can use the array syntax.
+
+```js
+import { reverseHighlightHit } from '@algolia/autocomplete-js';
+
+// An Algolia hit for query "hello"
+const hit = {
+  query: {
+    title: 'Hello there',
+  }
+  _highlightResult: {
+    query: {
+      title: {
+        value:
+          'Hello __aa-highlight__there__/aa-highlight__',
+      },
+    },
+  },
+};
+const reverseHighlightedValue = reverseHighlightHit({
+  hit,
+  attribute: ['query', 'title'],
```

Comment:
An example using this would make more sense:

```suggestion
  attribute: ['hierarchicalCategories', 'lvl1'],
```

Reply:
Updated both examples in 1f82cf29.